.devcontainer.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "Dev Container Definition - AWS CDK",
-    "image": "jsii/superchain",
+    "image": "mcr.microsoft.com/vscode/devcontainers/universal",
```

Comment:
is this change intentional? Seems not to be related to the PR's scope

Reply:
oh...my bad. I should not submit this.